package.json

```diff
@@ -42,7 +42,8 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^2.0.0",
-    "ember-data": "^3.0.2",
+    "ember-copy": "^1.0.0",
```

Comment:
I believe `ember-copy` should be a dependency instead of devDepencency. I tried to install this branch on my project and got errors saying it couldn't find `ember-copy`. I did an `npm install ember-copy` and that fixed it.